include/iarc7_fc_comms/MspFcComms.hpp

```diff
@@ -40,9 +40,13 @@ namespace FcComms
         FcCommsReturns  __attribute__((warn_unused_result))
             handleComms();
 
-        // Get the flight status of the FC.
+        // Find out if the FC is armed.
         FcCommsReturns  __attribute__((warn_unused_result))
-            getStatus(bool& armed, bool& auto_pilot, bool& failsafe);
+            isArmed(bool& armed);
+
+        // Find out if the FC is in failsafe
+        FcCommsReturns  __attribute__((warn_unused_result))
+            isFailsafe(bool& failsafe);
```

Comment:
This one also doesn't seem to be implemented, do we just want to leave the declaration until it is?